CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ To install or update CocoaPods see this [guide](http://docs.cocoapods.org/guides
 
 To install release candidates run `[sudo] gem install cocoapods --pre`
 
+## Master
+
+##### Bug Fixes
+
+* Add missing `.cc` extension for C++ to `SOURCE_FILE_EXTENSIONS`.
```

Comment:
@kylef Thanks for pointing this out. Updated via https://github.com/lembacon/CocoaPods/commit/43113ae1cca0351737d32076469548638abd8880.